### ISSUE TITLE
Missing include of QHeaderView in mainwindow.cpp on Qt5

### ISSUE THREAD

**Issue by rgov** [schulmar/Templar#13]
`gui/mainwindow.cpp` needs to `#include <QHeaderView.h>` to compile with Qt5.

**Comment by schulmar** [comment]
I cannot reproduce this issue. Which commit and which version of Qt5 are you using?

**Comment by rgov** [comment]
I was building the latest commit in the `feature/templight2` branch (or something like that). The version of Qt was 5.12.0, installed by Homebrew. My operating system is macOS 10.14.2.

It seems that some other Qt headers such as QTableView.h may have changed to use a forward declaration for QHeaderView, rather than import the header directly, and this code uses QHeaderView without explicitly importing.

Generally it's a good practice to import what the file is using, even if it may not cause a compiler error in some configurations.

**Comment by schulmar** [comment]
I could reproduce the problem on this branch. Your proposed fix is applied in db8fd94c28255fd9edbe0420d8234e3b30075464

> Generally it's a good practice to import what the file is using

I totally agree. Most of the code was not written by me and I did not have the time to do cleanups beyond getting it to compile and run. I am not even sure whether this tool is still compatible with templight2 as I did not use it for a very long time.
